networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/peers/ApprovedRequest.java

```diff
@@ -16,22 +16,26 @@
 import java.util.Objects;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
 
-public class RequestApproval {
+public class ApprovedRequest {
 
-  private final RequestsKey requestKey;
-  private final long objectsCount;
+  private final RequestKey requestKey;
+  private long requestSize;
 
-  private RequestApproval(final RequestsKey requestKey, final long objectsCount) {
+  private ApprovedRequest(final RequestKey requestKey, final long requestSize) {
```

Comment:
NIT: maybe it is me but `requestSize` makes me think about size in bytes instead of the limit of objects being requested.

The previous name also wasn't clear. Maybe something like `maxObjects` or `maxObjectsPerRequest` is clearer?

Reply:
will address this in a more complete way in a followup